requirements.txt

```diff
@@ -1,3 +1,2 @@
-torch>=2.3.0; sys_platform != 'win32'
-torch>=2.4.1; sys_platform == 'win32'
+torch>=2.4.1, <2.7.0
```

Comment:
@ericspod Was it intentional to also bump the minimum torch on non-Windows platforms from 2.3.0 to 2.4.1?